code/modules/admin/verbs/randomverbs.dm

```diff
@@ -658,13 +663,36 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	if(SSshuttle.emergency.mode >= SHUTTLE_DOCKED)
 		return
 
+	SSshuttle.emergencyNoRecall = 0
 	SSshuttle.emergency.cancel()
 	feedback_add_details("admin_verb","CCSHUT") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 	log_admin("[key_name(usr)] admin-recalled the emergency shuttle.")
 	message_admins("<span class='adminnotice'>[key_name_admin(usr)] admin-recalled the emergency shuttle.</span>")
 
+
 	return
 
+/client/proc/admin_disable_shuttle()
```

Comment:
I think you should check the mode of the shuttle before assigning to SHUTTLE_STRANDED. Perhaps check for IDLE first, then assign to STRANDED